ZAPDTR/ZAPD/ZResource.cpp

```diff
@@ -328,7 +328,7 @@ std::string ZResource::GetSourceOutputHeader([[maybe_unused]] const std::string&
 		std::string xmlPath = StringHelper::Replace(parent->GetXmlFilePath().string(), "\\", "/");
 
 		if (StringHelper::Contains(outName, "_room_") || StringHelper::Contains(outName, "_scene"))
-			prefix = "scenes";
+			prefix = "scenes/nonmq";
```

Comment:
All header files are created with the nonmq path. I figure that out of all the people playing with just one OTR, there will probably be more playing with a nonmq OTR so it makes sense for this to be the default. This is swapped out for `mq` when an MQ asset needs to be loaded.